cfe_internal/enterprise/federation/federation.cf

```diff
@@ -109,6 +116,8 @@ bundle agent config
       "I'm pulling dumps!";
     am_transporter.(DEBUG|DEBUG_config)::
       "I'm a transporter!";
+    am_paused.(DEBUG|DEBUG_config)::
```

Comment:
Should we look for default:DEBUG?

Reply:
So then it would be like (default:DEBUG|DEBUG_config)

And you could run `cf-agent --define DEBUG` OR `cf-agent --define cfengine_enterprise_federation:DEBUG_config`

Reply:
oh. yes! I agree!

Reply:
@nickanderson changed and added a longer comment as explanation. I suppose we might have a top-level place where we recommend the use of a DEBUG class? Because it doesn't directly get defined if you do `cf-agent --debug`.

Reply:
its an mpfism https://docs.cfengine.com/docs/3.12/guide-writing-and-serving-policy-policy-style.html#policy-reports